main.cpp

```diff
@@ -451,6 +560,18 @@ struct Stove
     void boilWater(int setLevel = 9);
     void frySteaks(int setLevel = 7);
     void makeSoup(int setLevel = 5);
+    int cookBigMeal(int numPotsNeeded = 5, int inUse = 0)
+    {
+        hotplatesInUse = inUse;
+        while(hotplatesInUse < numberHotplates && numPotsNeeded > 0)
+        {
+            numPotsNeeded --;
+            hotplatesInUse ++;
```

Comment:
use `pre-inc` and `pre-dec` instead of `post-inc` and `post-dec`
https://en.cppreference.com/w/cpp/language/operator_incdec
- [ ] revise